lib/hexpm/accounts/users.ex

```diff
@@ -17,7 +17,10 @@ defmodule Hexpm.Accounts.Users do
   end
 
   def put_repositories(user) do
-    repositories = Map.new(user.repositories, &{&1.id, &1})
+    repositories =
+      user.repositories
+      |> Map.new(&{&1.id, &1})
+      |> Map.put(1, Repository.hexpm())
```

Comment:
Ah crap. Now I see why you needed to do the special `repository_id` check and I think we need to revert back to it.

I think this change will break `mix hex.user auth` because we try to auth all organizations the API says the user is member of.

Reply:
Actually we use `User.repository_users` but I think `User.repositories` and `User.repository_users` should have the same repositories.